tox.ini

```diff
@@ -54,108 +54,123 @@ envlist =
     python-framework_django-{py37,py38,py39,py310,py311}-Django0302,
-    python-framework_falcon-{py37,py38,py39,pypy37}-falcon0103,
+    python-framework_falcon-{py27,py37,py38,py39,pypy,pypy37}-falcon0103,
     python-framework_falcon-{py37,py38,py39,py310,pypy37}-falcon{0200,master},
     # Falcon master branch failing on 3.11 currently.
     python-framework_falcon-py311-falcon0200,
     python-framework_fastapi-{py37,py38,py39,py310,py311},
-    python-framework_flask-{py37,py38,py39,py310,py311,pypy37}-flask0101,
+    python-framework_flask-{pypy,py27}-flask0012,
+    python-framework_flask-{pypy,py27,py37,py38,py39,py310,py311,pypy37}-flask0101,
     ; temporarily disabling flaskmaster tests
     python-framework_flask-{py37,py38,py39,py310,py311,pypy37}-flask{latest},
     python-framework_graphene-{py37,py38,py39,py310,py311}-graphenelatest,
-    python-framework_graphene-{py37,py38,py39,pypy37}-graphene{0200,0201},
+    python-framework_graphene-{py27,py37,py38,py39,pypy,pypy37}-graphene{0200,0201},
     python-framework_graphene-{py310,py311}-graphene0201,
-    python-framework_graphql-{py37,py38,py39,py310,py311,pypy37}-graphql02,
+    python-framework_graphql-{py27,py37,py38,py39,py310,py311,pypy,pypy37}-graphql02,
     python-framework_graphql-{py37,py38,py39,py310,py311,pypy37}-graphql03,
     ; temporarily disabling graphqlmaster tests
     python-framework_graphql-py37-graphql{0202,0203,0300,0301,0302},
+    grpc-framework_grpc-py27-grpc0125,
     grpc-framework_grpc-{py37,py38,py39,py310,py311}-grpclatest,
-    python-framework_pyramid-{py38}-Pyramid0104,
-    python-framework_pyramid-{pypy37,py37,py38,py39,py310,py311}-Pyramid0110-cornice,
+    python-framework_pyramid-{pypy,py27,py38}-Pyramid0104,
+    python-framework_pyramid-{pypy,py27,pypy37,py37,py38,py39,py310,py311}-Pyramid0110-cornice,
     python-framework_pyramid-{py37,py38,py39,py310,py311,pypy37}-Pyramidlatest,
     python-framework_sanic-{py38,pypy37}-sanic{190301,1906,1912,200904,210300,2109,2112,2203,2290},
     python-framework_sanic-{py37,py38,py39,py310,py311,pypy37}-saniclatest,
     python-framework_starlette-{py310,pypy37}-starlette{0014,0015,0019},
     python-framework_starlette-{py37,py38}-starlette{002001},
     python-framework_starlette-{py37,py38,py39,py310,py311,pypy37}-starlettelatest,
     python-framework_strawberry-{py37,py38,py39,py310,py311}-strawberrylatest,
-    python-logger_logging-{py37,py38,py39,py310,py311,pypy37},
+    python-logger_logging-{py27,py37,py38,py39,py310,py311,pypy,pypy37},
     python-logger_loguru-{py37,py38,py39,py310,py311,pypy37}-logurulatest,
     python-logger_loguru-py39-loguru{06,05,04,03},
-    libcurl-framework_tornado-{py37,py38,py39,py310,py311,pypy37}-tornado0600,
-    libcurl-framework_tornado-{py38,py39,py310,py311}-tornadomaster,
-    rabbitmq-messagebroker_pika-{py37,py38,py39,pypy37}-pika0.13,
+    python-framework_tornado-{py37,py38,py39,py310,py311,pypy37}-tornado0600,
```

Comment:
I was a little confused about this change at first but I guess we're just standardizing this to be python-framework_ like the others.

Reply:
Yeah that's the idea. No need to install packages on the fly when they can be baked in.